crates/turborepo-lib/src/query/file.rs

```diff
@@ -147,11 +154,21 @@ impl File {
         Ok(self.path.to_string())
     }
 
-    async fn dependencies(&self, depth: Option<usize>) -> TraceResult {
+    async fn dependencies(&self, depth: Option<usize>, ts_config: Option<String>) -> TraceResult {
```

Comment:
I think this is a behavior change where we now respect `tsconfig`? Maybe update PR description to include functionality this gains us? (I'm curious and want to know, but can't grok it from this PR)